irohad/main/irohad.cpp

```diff
@@ -146,7 +146,14 @@ int main(int argc, char *argv[]) {
     irohad.storage->insertBlock(block.value());
     log->info("Genesis block inserted, number of transactions: {}",
               block.value().transactions.size());
+  } else {
+    // Recover VSW from the existing ledger to be sure it is consistent
+    if (not irohad.storage->recoverWSV()) {
+      log->error("Failed to recover WSV");
```

Comment:
At the moment, error is displayed at least two times - `match()` in `recoverWSV`, and here. Having a result and `match()` call here will be more consistent for logging.